src/Parser.jl

```diff
@@ -3,12 +3,13 @@ module Parser #JSON
 using Compat
 
 #Define ordered dictionary from DataStructures if present
-_HAVE_DATASTRUCTURES = try
-    using DataStructures
-    import DataStructures.OrderedDict
-    true
-catch
-    false
+function __init__()
+    global _HAVE_DATASTRUCTURES = try
```

Comment:
I'm going to merge this then, fix what's broken. Adjusting constness can be done separately if it makes a difference.